apps/devhub/tests/test_views_versions.py

```diff
@@ -186,6 +187,24 @@ def test_user_can_disable_addon(self, hide_mock):
         msg = entry.to_string()
         assert self.addon.name.__unicode__() in msg, ("Unexpected: %r" % msg)
 
+    @mock.patch('devhub.views.unindex_addons')
+    def test_user_can_unlist_addon(self, unindex):
+        self.addon.update(status=amo.STATUS_PUBLIC, disabled_by_user=False,
+                          is_listed=True)
+        res = self.client.post(self.unlist_url)
+        self.assertEqual(res.status_code, 302)
```

Comment:
I'm not against those, it's just that they're often useless like the `self.assertEqual` that doesn't do more, and often less, than py.test's `assert x == y`, and I find them un-pythonic (camelcase? really?), often more verbose, and tie the tests to their class.

The last point is moot because we're not going to rewrite all our tests to be simple functions using pytest fixtures.

Some are handy, like `assertTemplateUsed`, or `assertFormError`, but they're rarely used.

To sum it up: I think pytest's assertions introspections are more powerful (see the message you have when comparing two dicts or tuples, telling you which elements are missing or added, or which keys have different values...), and they look more pythonic, that's why I try to use them whenever I can.

What do you think?